--- PKGBUILD	2023-12-02 22:43:38.667492697 -0800
+++ PKGBUILD	2023-12-02 22:44:14.417567456 -0800
@@ -6,2 +6 @@
-pkgname=(oxygen
-         oxygen5)
+pkgname=(oxygen)
@@ -38,2 +38 @@
-source=(git+https://commits.kde.org/${pkgname}
-        qt-6.7.patch)
+source=(git+https://commits.kde.org/${pkgname})
@@ -48,4 +48,0 @@
-
-prepare() {
-  patch -d $pkgname -p1 < qt-6.7.patch # Fix build with Qt 6.7
-}
@@ -63,24 +61,0 @@
-package_oxygen5() {
-  pkgdesc='Qt5 Oxygen style'
-  depends=(oxygen
-           gcc-libs
-           glibc
-           frameworkintegration5
-           kcompletion5
-           kconfig5
-           kconfigwidgets5
-           kguiaddons5
-           ki18n5
-           kwidgetsaddons5
-           kwindowsystem5
-           libxcb
-           qt5-base
-           qt5-declarative
-           qt5-x11extras)
-  conflicts=('oxygen<5.27.80')
-  replaces=('oxygen<5.27.80')
-  groups=()
-
-  DESTDIR="$pkgdir" cmake --install build5
-  rm -r "$pkgdir"/usr/share/kstyle
-}
